packages/block-editor/src/components/block-alignment-matrix-toolbar/README.md

```diff
@@ -0,0 +1,73 @@
+# Alignment Matrix Toolbar
+The alignment matrix allows users the ability to change text alignment quickly and intuitively.
```

Comment:
i think we should disambiguate between "text alignment" using this component and "text alignment" using AlignmentToolbar component.

Reply:
This component (for Toolbar) aligns the inner blocks - not specific to text :)